scripts/merge-pr.py

```diff
@@ -310,12 +310,6 @@ def fix_version_from_branch(branch, versions):
 
 print("\n=== Pull Request #%s ===" % pr_num)
 
-# we may have un-printable unicode in our title
-try:
-    title = title.encode("raw_unicode_escape")
-except Exception:
-    pass
-
```

Comment:
@pandas-dev/pandas-core : Out of curiosity, how many people actually use this nowadays?

Reply:
Never used it, happy to get rid of it if nobody does.